filters/auth/tokeninfocache_test.go

```diff
@@ -111,7 +118,7 @@ func TestTokeninfoCache(t *testing.T) {
 
 	assert.Equal(t, int32(1), authRequests, "expected no request to auth sever")
 	assert.Equal(t, token, info["uid"])
-	assert.Equal(t, float64(595), info["expires_in"], "expected TokenTTLSeconds - truncate(delay)")
```

Comment:
It is still quite not obvious why 595 changed to 594?

Is it related to this part of code?
https://github.com/zalando/skipper/pull/3267/files#diff-a2721f4aa66d24557c036d686d4e3344636871b2de93fb75266b8da83d511891L79

Reply:
Yes, previous version adjusted cached `expires_in`  based on the time elapsed since entry was cached.
Since the number must be integer it truncated the elapsed time:
```
new_expires_in = expires_in - truncateToSeconds(now-cachedAt)
```
this version stores expiration date instead of cached date and calculates expires_in from it:
```
new_expires_in = truncateToSeconds(infoExpiresAt-now)
```

This test moves clock by `5.7` seconds so previous version resulted in:
```
new_expires_in = expires_in - truncateToSeconds(now-cachedAt) = 600 - truncateToSeconds(cachedAt+5.7-cachedAt) =  truncateToSeconds(5.7) = 595
```
and this version results in:
```
new_expires_in = truncateToSeconds(infoExpiresAt-now) = truncateToSeconds(cachedAt+600-(cachedAt+5.7)) = truncateToSeconds(594.3) = 594
```